cypress/page/login.page.ts

```diff
@@ -0,0 +1,28 @@
+class LoginPage{
+
+    private email: string;
```

Comment:
could you add to the attributes of each page object the type of element we are interacting with? for example in this case, the element is an input